app/Keycloak/Config.php

```diff
@@ -6,24 +6,9 @@
 
 final readonly class Config
```

Comment:
I no longer see a need for this class. I would directly access the feature flag like we do for Auth0 and UiTiD and in other places use the handy RealmCollection.